code/src/Styles/Globalstyles.js

```diff
@@ -0,0 +1,111 @@
+// /////////////// IMPORTS //////////////////////// //
+
+import styled, { createGlobalStyle } from 'styled-components/macro';
+
+// /////////////// GLOBAL //////////////////////// //
+
+export const GlobalRules = createGlobalStyle`
```

Comment:
I saw that you hade these rendered in the App-component. I've never seen that before? What would be the benefit compared to having these rules in index.css? (Sorry for a lot of questions hahah! I'm just very curious!)